### ISSUE TITLE
Create from a ROS project template

### ISSUE THREAD

**Issue by jingxiangmo** [jingxiangmo/create-ros-project#2]

**Comment by TheBigSasha** [comment]
Would be very nice to have:
- Some template projects like a small car, walker, quadcopter
- Templates would include a gazebo simulator pre configured so that a user can skip right to the actual development


A measure of success:
Can we replace this guide with just one command and then RL work similarly to setting up and using Gym? http://wiki.ros.org/reinforcement_learning/Tutorials/Reinforcement%20Learning%20Tutorial

**Comment by jingxiangmo** [comment]
yeah that's a great idea. Could make it easy for RL researchers to get started with Gazebo
